azure-pipelines.yml

```diff
@@ -11,34 +11,82 @@ pr:
       - develop
   drafts: false
 
-pool:
-  vmImage: ubuntu-latest
-
-steps:
-- task: UseDotNet@2
-  displayName: 'Install DotNet'
-  inputs:
-    packageType: 'sdk'
-    version: '3.x'
-
-- task: DotNetCoreCLI@2
-  displayName: 'DotNet Build'
-  inputs:
-    command: 'build'
-    projects: $(projects)
-    arguments: '--output $(Build.ArtifactStagingDirectory) $(styleCopEnabled)'
-
-- task: DotNetCoreCLI@2
-  displayName: 'DotNet Tests'
-  inputs:
-    command: test
-    projects:  $(testsProjects)
-
-- task: PublishBuildArtifacts@1
-  displayName: 'DotNet Drop'
-  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'))
-  inputs:
-    PathtoPublish: '$(Build.ArtifactStagingDirectory)'
-    ArtifactName: 'Drop'
-    publishLocation: 'Container'
-  
\ No newline at end of file
+variables:
+  projects: '**/*.sln'
+  azureSubscription: 'devlevel3-dev (a797242d-ce60-41aa-98ca-2e22d5221cfe)'
+  functionAppName: 'fa-tuapi-resumes-dev-bra'
+  vmImageName: 'ubuntu-latest'
+
```

Comment:
we don't want variables for this pipeline, please remove all and keep the values, we'll add them through Azure DevOps UI

Reply:
It's done

Reply:
no, it's not done, I wanted them to be added trough Az UI, but now you hardcoded them/
it's fine, let's iterate to enhance